src/Place/PlaceEditingServiceInterface.php

```diff
@@ -141,7 +141,7 @@ public function addImage($id, UUID $imageId);
     /**
      * Update an image of the place.
      *
-     * @param $id
+     * @param string $id
```

Comment:
None of the methods in this class has this. It would require a cascade of changes to the OfferEditingServiceInterface and all its implementations as well. That's why I only changed the docblock for now.